docs/blog/newsletter-july-2022.rst

```diff
@@ -0,0 +1,118 @@
+
+Ranging further afield, folks pointed to documentation's crucial work in defining audience, creating a single source of truth for the information products need to help their users, and establishing a role in designing and building the products themselves. Perhaps one contributor summarized it best when they said that technical writing "is a diverse field that pragmatically applies theories from other fields as they make sense in the situation." In other words, we look to theory that's informed by practice, rather than the other way around.
+
+Toward the end of the wide-ranging conversation, folks dug into the question of documentation's role in persuading users (in addition to informing and instructing). While some pointed out that this role is conventionally associated with marketing and that marketing language tends to get in the way of the primary goals of informing and instructing, the consensus was that good documentation can and does persuade users to either start with a product or continue with it. Documentation does all the things! ... which might be part of why it's so difficult to provide it with a useful theoretical foundation.
+
+------------------
+What we’re reading
+------------------
+
+The #bipoc group’s been discussing the following materials on diversity, inclusion, and equity. Want to join the conversation? Please join us in the `#bipoc Slack channel <https://writethedocs.slack.com/archives/C016STMEWJD>`_!
+
+A short read: Google, and other big tech companies, are struggling with `caste discrimination <https://www.nbcnews.com/news/asian-america/big-techs-big-problem-also-best-kept-secret-caste-discrimination-rcna33692>`_ after cancelling a Dalit History Month talk.
+
+A medium read: Belonging to a community of support can help shape your career and your life. However, for Black people, it can be hard to find role models in the tech field. The New Stack writes about `BlackTechTwitter <https://thenewstack.io/blacktechtwitter-for-life/>`_ and includes a list of accounts to follow.
+
+A longer read: Textio released their `Language Bias in Performance Feedback report <https://www.oprahdaily.com/life/a32893726/what-is-juneteenth/>`_. They found that when looking at job performance feedback, patterns of bias appear for most all underrepresented groups.
+
+----------------
+From our sponsor
```

Comment:
Just to confirm this will get filled in before shipping

Reply:
Yep. Just did :slightly_smiling_face: